package.json

```diff
@@ -47,6 +47,7 @@
     "@babel/core": "^7.20.2",
     "@babel/preset-env": "^7.20.2",
     "@babel/register": "^7.18.9",
+    "@gulpjs/gulplog-v1": "npm:gulplog@1.0.0",
```

Comment:
I'm fine with this because I'm mapping to the `@gulpjs` scope and this is only a devDep for the tests